src/rebuild/srv.c

```diff
@@ -176,7 +176,11 @@ rebuild_leader_set_status(struct rebuild_global_pool_tracker *rgt,
 		}
 	}
 
-	D_ASSERTF(status != NULL, "Can not find rank %u\n", rank);
+	if (status == NULL) {
+		D_INFO("rank %u is not included in this rebuild.\n", rank);
+		return;
```

Comment:
"If the rank is being added to the pool map after rebuild starts"
is this only possible for pool upgrade triggered rebuild?